### ISSUE TITLE
How to use icon with fill=1?

### ISSUE THREAD

**Issue by halafi** [marella/material-symbols#30]
is it possible to use icons with fill=1 as shown in demo https://marella.me/material-symbols/demo/#outlined ?

**Comment by Firescar96** [comment]
Yes, instruction here: https://www.npmjs.com/package/material-symbols#usage

**Comment by Firescar96** [comment]
Yes, it's described here: https://www.npmjs.com/package/material-symbols#usage

**Comment by c0d3x** [comment]
Also have issues with filled icons:
I tried using scss mixins for adding filled icons to btn classes, but for me it does not work with filled icons using only css.
It only managed to make it work with a react component that use an icon component that has the flag on render.
